solutions/landing-zone/README.md

```diff
@@ -53,7 +53,7 @@ The following resources will be deployed.
 | ---- | ---- | ---- | --- | -- |
 | Access Context Manager | | | Work In Progress | `environments/common/vpc-service-controls`
 | Core Audit Bunker | | | Available | `environments/common/audit` |
-| Core Folders | | | Available | `environments/common/hiearchy.yaml` |
```

Comment:
Alain, thanks for testing the folder option and finding the hierarchy issue as a result
every snippet of code in non-executable readme's needs to be verified/run - in unit or ideally in a clean LZ deployment run.